tensorflow/core/kernels/mkl_concat_op.cc

```diff
@@ -697,7 +765,11 @@ class MklConcatOp : public OpKernel {
       if (isMklReorderNeeded) {
         for (int k = 0; k < input_tensors.size(); k++) {
           if (input_tensors[k].NumElements() > 0) {
+#ifdef ENABLE_MKLDNN_V1
+            srcs[k].CheckReorderToOpMem(srcs_pd[k], cpu_engine);
+#else
             srcs[k].CheckReorderToOpMem(srcs_pd[k]);
+#endif  // ENABLE_MKLDNN_V1
```

Comment:
We can use
```c++
srcs[k].CheckReorderToOpMem(MEMORY_PD_WITHOUT_DATA(srcs_pd[k], cpu_engine));
```
(The naming doesn't really make sense here, but essentially the macro just drops `cpu_engine` when it's not v1. We should probably rename the macro later.)

Reply:
ok thank you